app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  #  :lockable, :timeoutable,  and :omniauthable
+  devise :database_authenticatable, :registerable, :confirmable, :trackable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :categories
+  has_many :transactions
+end
```

Comment:
- [ ] Also here, as suggested earlier we need to make sure the Categories section is relatively the same as the original template. As you can see in your case it's a bit different from the expected design. Please Consider updating it to match the given design. 

- [Optional] Also, I noticed you have an already defined Icon name for the category. This is not a good idea as we are looking for a way to implement an option for the user to enter an image URL that he/she expects to match the type of category.

### Your design 
<img width="383" alt="Screenshot 2023-11-23 at 4 31 00 in the afternoon" src="https://github.com/wolterbwambale/Budget-app/assets/69077061/722bac1f-3f28-4a33-a8f1-423176e409fb">
### Example for the expected design 

![](https://user-images.githubusercontent.com/58642949/285068776-feeb05a0-fcbe-43ca-9887-25d8c1cdaaf6.png)